mobile-widgets/qmlmanager.cpp

```diff
@@ -526,7 +526,7 @@ void QMLManager::saveCloudCredentials(const QString &newEmail, const QString &ne
 	if (qPrefCloudStorage::cloud_verification_status() != qPrefCloudStorage::CS_NOCLOUD &&
 		!cloudCredentialsChanged) {
 		// just go back to the dive list
-		QMLPrefs::instance()->setCredentialStatus(QMLPrefs::instance()->oldStatus());
+		qPrefCloudStorage::set_cloud_verification_status(QMLPrefs::instance()->oldStatus());
```

Comment:
the commit message tells me that you are `setting oldStatus() directly to qPrefCloudStorage`
This may be a language issue, but based on that message I would assume code that says
`oldStatus = qPrefCloudStorage`
which is of course not what you are trying to tell me.
Did you mean something like `restore the previous state in qPrefCloudStorage instead of QMLPrefs` and then continue on to explain why that is the right thing to do?

Reply:
refrased